src/main/java/org/breedinginsight/daos/ProgramDAO.java

```diff
@@ -124,20 +147,50 @@ public ProgramDAO(Configuration config, DSLContext dsl, BrAPIProvider brAPIProvi
         return resultPrograms;
     }
 
-    public ProgramBrAPIEndpoints getProgramBrAPIEndpoints() {
-        return new ProgramBrAPIEndpoints();
+    public int getNumProgramUsers(UUID programId) {
+        return dsl.selectCount().from(PROGRAM_USER_ROLE)
+                .where(PROGRAM_USER_ROLE.PROGRAM_ID.eq(programId)
+                        .and(PROGRAM_USER_ROLE.ACTIVE.eq(true)))
+                .fetchOne(0, Integer.class);
     }
 
     public ProgramBrAPIEndpoints getProgramBrAPIEndpoints(UUID programId) {
-        // Just returns defaults for now
-        // TODO: in future return urls for given program
+        ProgramEntity programEntity = fetchOneById(programId);
+
+        String coreUrl = defaultBrAPICoreUrl;
+        String genoUrl = defaultBrAPIGenoUrl;
+        String phenoUrl = defaultBrAPIPhenoUrl;
+
+        // only storing one program brapi url for now so set all to that one
+        if (!StringUtils.isBlank(programEntity.getBrapiUrl())) {
+            String brapiUrl = programEntity.getBrapiUrl();
+            coreUrl = brapiUrl;
+            genoUrl = brapiUrl;
+            phenoUrl = brapiUrl;
+        }
+
```

Comment:
Looks like there is just one url for the program now. Should we reflect that in this function? Or are we keeping it this way in case we want to have multiple urls in the future? If we want to change it, maybe we can create a new card so it doesn't inflate this feature?

Reply:
Kept it because eventually we might want to have the ability to set these independently and not just one program URL.